lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -43,6 +43,11 @@ def call(env)
           # retrieve integration settings
           tracer = configuration[:tracer]
 
+          if configuration[:distributed_tracing]
```

Comment:
Can we leave a comment here on why it needs to happen here instead of down below?

Reply:
Sure, no problem.